internal/cmd/rsl/record/record.go

```diff
@@ -24,6 +25,12 @@ func (o *options) AddFlags(cmd *cobra.Command) {
 	)
 }
 
+func (o *options) CheckSigningKeys(_ *cobra.Command, arg []string) error {
+	_, _, err := gitinterface.GetSigningCommand()
```

Comment:
I'd make a generic "is signing viable" function that uses this and put it in cmd/common. Then we can also plug this into the `trust` and `policy` commands that also record RSL entries but don't use this exact command.